Emby.Server.Implementations/StartupOptions.cs

```diff
@@ -1,30 +1,43 @@
-using System;
-using System.Linq;
-
 namespace Emby.Server.Implementations
 {
+    using CommandLine;
+
+    /// <summary>
+    /// Class used by CommandLine package when parsing the command line arguments.
+    /// </summary>
     public class StartupOptions
     {
-        private readonly string[] _options;
+        [Option('d', "datadir", Required = false, HelpText = "Path to use for the data folder (databases files etc.).")]
+        public string PathData { get; set; }
```

Comment:
Ok, so this one is confusing since there is a dir `data` inside this one.

Reply:
Yes bit confusing having that 'data' folder under -programdatadir.  What to do?  Rename this to DataDir?

Reply:
I think the solution is to rename the 'data' folder under DataDir to something else which better reflects its contents.  'database' might be better but that doesn't cover all contents.  May I suggest that improvement is not handled under this PR please?